py3-snowballstemmer.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: py3-snowballstemmer
+  version: 2.2.0
+  epoch: 0
+  description: This package provides 29 stemmers for 28 languages generated from Snowball algorithms.
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+      - python3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python3
+      - py3-setuptools
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 09b16deb8547d3412ad7b590689584cd0fe25ec8db3be37788be3810cbf19cb1
+      uri: https://files.pythonhosted.org/packages/44/7b/af302bebf22c749c56c9c3e8ae13190b5b5db37a33d9068652e8f73b7089/snowballstemmer-${{package.version}}.tar.gz
```

Comment:
can this also be a `git-checkout` https://github.com/snowballstem/snowball/releases/tag/v2.2.0?

Reply:
I tried but the "setup.py" does not exist on GitHub source but it exists on files.pythonhosted source, so, I couldn't understand how to build with by using GitHub source

Reply:
Lets stick with the tarball for now.